src/shared/behaviors/validatable.html

```diff
@@ -63,7 +63,7 @@
 				});
 				return result.length === this.testSet.length;
 			} else {
-				return false;
+				return true;
```

Comment:
how exposed is this API?  maybe consider changing the method name to `isValid` or `isValueValid` (somewhat less ambiguous about what the result/return-value means) if it won't require a full point-release due to breaking interface.